Source/SwiftLintFramework/Rules/RuleConfigurations/InclusiveLanguageConfiguration.swift

```diff
@@ -0,0 +1,49 @@
+private enum ConfigurationKey: String {
+    case severity
+    case allow
+    case deny
+}
+
+public struct InclusiveLanguageConfiguration: RuleConfiguration, Equatable {
+    public var severityConfiguration = SeverityConfiguration(.warning)
+    public var denyList: Set<String>
+
+    public var consoleDescription: String {
+        severityConfiguration.consoleDescription
+            + ", deny: \(denyList.sorted())"
+    }
+
+    public var severity: ViolationSeverity {
+        severityConfiguration.severity
+    }
+
+    private let defaultDenyList: Set<String> = [
+        "whitelist",
+        "blacklist",
+        "master",
+        "slave"
+    ]
+
+    public init() {
+        self.denyList = defaultDenyList
+    }
+
+    public mutating func apply(configuration: Any) throws {
+        guard let configuration = configuration as? [String: Any] else {
+            throw ConfigurationError.unknownConfiguration
+        }
+
+        if let severityString = configuration[ConfigurationKey.severity.rawValue] {
+            try severityConfiguration.apply(configuration: severityString)
+        }
+
+        let configAllowList = lowercasedList(for: .allow, from: configuration)
+        let configDenyList = lowercasedList(for: .deny, from: configuration)
+        self.denyList = defaultDenyList.union(configDenyList).subtracting(configAllowList)
```

Comment:
is the idea that people should use `deny` to add more cases and `allow` to remove from the default ones?

I wonder if we should make the configuration like `additional_terms` and `terms` where you could use the latter to ignore the default list and setup yours completely

Reply:
Yah, that's the idea. I'm a bit worried that `terms` could be more ambiguous than `allow`/`deny`. `allow` means you're allowed to use the term in your project, and `deny` means you're not allowed. So, if the default list contains terms you want to use, you'd need to add them to `allow`. In your example, would the mere presence of the `terms` key cause the default list to be bypassed? I think we'd want to make sure this was super clear to users. I think I could be convinced either way.

Reply:
Maybe `override_terms`? 

> In your example, would the mere presence of the terms key cause the default list to be bypassed? 

Yeah, that's what I was thinking